.github/workflows/canary-integration-test.yml

```diff
@@ -47,6 +47,11 @@ jobs:
         GOPATH=$(go env GOPATH) make clean && make -j$nproc IMAGES='ceph' VERSION=0 build
         docker images
         docker tag $(docker images|awk '/build-/ {print $1}') rook/ceph:master
+    
+    - name: validate yaml 
+      run: |
+        cd cluster/examples/kubernetes/ceph/
```

Comment:
there is a yamllint tool https://pypi.org/project/yamllint  available you want to explore also?

Reply:
I looked to this but here we want to just validate the schema which `-dry-run` will do but yes if want we can add linter too.
 @travisn ?

Reply:
Done.

Reply:
I know we have a lot of actions that we can run, but isn't it still better if we don't use too many actions at once for each PR? Is there a reason why we wouldn't just have a single YAML Lint action that runs both linters in different steps?

Reply:
The dry run action only takes about 2s if the minikube cluster is already up, so in that case I thought it made sense to add it to the existing minikube action instead of requiring this action to fully bring up minikube.